src/Controls/tests/Core.UnitTests/BindableLayoutTests.cs

```diff
@@ -627,13 +627,13 @@ public async Task DoesNotLeak()
 
 			// First GC
 			await Task.Yield();
-			GC.Collect();
+			GC.Collect(2);
 			GC.WaitForPendingFinalizers();
```

Comment:
Since there are many permutations of code like this littered thoughout the tests.

Can we make a single `TestUtilities.cs` file we can share between all the test projects? You shouldn't have to update so many files to introduce the number `2`.